helpdesk_mgmt_sla_rule/models/helpdesk_sla.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class HelpDeskTicket(models.Model):
+    _inherit = "helpdesk.sla"
+
+    def check_sla(self):
+        tickets = self.env["helpdesk.ticket"].search([("sla_id", "!=", False)])
```

Comment:
1. add comment to explain why this is a full override of the method
2. you can't currently search on this field because the `sla_id` field is not stored